subsys/bluetooth/shell/bt.c

```diff
@@ -1652,26 +1652,33 @@ static int cmd_per_adv_sync_delete(const struct shell *shell, size_t argc,
 				   char *argv[])
 {
 	int err;
-	int index = 0;
-	struct bt_le_per_adv_sync **per_adv_sync = NULL;
+	int index;
+	struct bt_le_per_adv_sync *per_adv_sync = NULL;
```

Comment:
nitpick, reverse Christmas tree declarations.

Reply:
I fixed that, but I might not in the future as A) this hardly followed in any (host) file for BT and B) this isn't declared in the coding style guide. Can you point me to any place that states that we should use that format when declaring variables? The only place I can find is https://github.com/zephyrproject-rtos/zephyr/issues/2248 which seems slightly outdated.

Reply:
BT files like hci_core.c, conn.c etc. did try follow it before. Reverse Christmas tree was requested when initially contributing the controller code. Yes, not all subsystems follow it in Linux either but some do. It is not a strict rule, and some tried patches to checkpatch.pl (https://lore.kernel.org/patchwork/patch/732076/)

Also, https://github.com/zephyrproject-rtos/zephyr/pull/21184#discussion_r353831687 but what got merged was entirely the opposite.